frontend/utils.ts

```diff
@@ -124,9 +124,9 @@ export const toHoleBigInt = (numb: string, decimal: number) => {
     for (let index = 0; index < decimal; index++) {
       addZeros = "0" + addZeros;
     }
-    return BigInt(parts[0] + addZeros);
+    return BigInt(parts[0].replace(/,/g, "") + addZeros);
```

Comment:
the , means the 1,000? or means 1000,00?

Reply:
1,000

Reply:
ok. it works with big numbers? should be replaceAll?